sample_util.js

```diff
@@ -5,15 +5,13 @@
  */
 'use strict';
 
-const msRestAzure = require('ms-rest-azure');
-const azureConstants = require('ms-rest-azure/lib/Constants');
-const AzureEnvironment = require('ms-rest-azure/lib/AzureEnvironment');
-const KeyVault = require('azure-keyvault');
-const KeyVaultManagementClient = require('azure-arm-keyvault');
-const ResourceManagementClient = require('azure-arm-resource').ResourceManagementClient;
+const util = require('util');
+const { KeyVaultManagementClient } = require('@azure/arm-keyvault');
+const { ResourceManagementClient } = require('@azure/arm-resources');
 const random_id = require('./random_id');
 const adal = require('adal-node');
-const request = require('request-promise-native');
+const { ClientSecretCredential } = require('@azure/identity');
```

Comment:
Please change ClientSecretCredential to DefaultAzureCredential as service team's requirement. Thanks